### PR TITLE
chore(react-tags-preview): update dismiss and overflow story

### LINKED ISSUE #29220: TagGroup overflow example needs more padding
TagGroup uses `column-gap` to set the gap between tags. The overflow library won't handle this very well until  #23066

For now it's probably OK to increase the `padding` on the `Overflow` component to 60px

image

### LINKED ISSUE #29221: Tag dismiss examples should have a 'reset' button
image

Once tags are dismissed in docs they are gone until the user switches to another page or refreshes the page. It would be nice to have a 'reset' button to add tags back again

### PR DESCRIPTION
**Pull request by YuanboXue-Amber**
fix #29221 - add reset button to dismiss example:
Screenshot 2023-09-22 at 13 02 04
When all tags are dismissed, the dismiss button is focused+enabled: 
Screenshot 2023-09-22 at 13 02 14

fix #29220 - increase overflow example padding

### CODE REVIEW

**Comment by fabricteam** [comment]
## 📊 Bundle size report

<details>
<summary>Unchanged fixtures</summary>

| Package & Exports                                                                                                |         Size (minified/GZIP) |
| ---------------------------------------------------------------------------------------------------------------- | ---------------------------: |
| <samp>react-tags-preview</samp> <br /> <abbr title='bundle-size/InteractionTag.fixture.js'>InteractionTag</abbr> |   `14.23 kB`<br />`5.722 kB` |
| <samp>react-tags-preview</samp> <br /> <abbr title='bundle-size/tag.fixture.js'>Tag</abbr>                       |  `29.985 kB`<br />`9.667 kB` |
| <samp>react-tags-preview</samp> <br /> <abbr title='bundle-size/TagGroup.fixture.js'>TagGroup</abbr>             | `72.871 kB`<br />`21.705 kB` |

</details>
<sub>🤖 This report was generated against <a href='https://github.com/microsoft/fluentui/commit/01d9fdb588ee13cd881e9c053d13e8b8e54b3bcf'>01d9fdb588ee13cd881e9c053d13e8b8e54b3bcf</a></sub>
<div id="bundleSizeComment9423"/>

**Comment by fabricteam** [comment]
## 🕵 fluentuiv9 No visual regressions between this PR and main
<div id="vrtCommentfluent-ui_VRT_Pipeline_v9"/>

**Review by bsunderhus — APPROVED** [review]

File: packages/react-components/react-tags-preview/stories/InteractionTag/InteractionTagDismiss.stories.tsx [hunk 1]
```diff
@@ -8,37 +8,86 @@ import {
...
+            );
+          })}
+        </TagGroup>
+      )}
+
+      <Button
+        onClick={resetItems}
+        ref={resetButtonRef}
+        disabled={visibleTags.length !== 0}
+        className={styles.resetButton}
```
Comment (layershifter) [inline packages/react-components/react-tags-preview/stories/InteractionTag/InteractionTagDismiss.stories.tsx:86]:
```suggestion
        className={styles.resetButton}
        size="small"
```

nit: to make it clear that it's not a part of the example

File: packages/react-components/react-tags-preview/stories/InteractionTag/InteractionTagDismiss.stories.tsx [hunk 2]
```diff
@@ -8,37 +8,86 @@ import {
...
+      >
+        Reset Dismiss Example
```
Comment (layershifter) [inline packages/react-components/react-tags-preview/stories/InteractionTag/InteractionTagDismiss.stories.tsx:88]:
```suggestion
        Reset the example
```

File: packages/react-components/react-tags-preview/stories/Tag/TagDismiss.stories.tsx [hunk 3]
```diff
@@ -1,26 +1,82 @@
...
+        </TagGroup>
+      )}
+
+      <Button
+        onClick={resetItems}
+        ref={resetButtonRef}
+        disabled={visibleTags.length !== 0}
+        className={styles.resetButton}
+      >
+        Reset Dismiss Example
```
Comment (layershifter) [inline packages/react-components/react-tags-preview/stories/Tag/TagDismiss.stories.tsx:77]:
```suggestion
        Reset the example
```

File: packages/react-components/react-tags-preview/stories/Tag/TagDismiss.stories.tsx [hunk 4]
```diff
@@ -1,26 +1,82 @@
...
+            </Tag>
+          ))}
+        </TagGroup>
+      )}
+
+      <Button
+        onClick={resetItems}
+        ref={resetButtonRef}
+        disabled={visibleTags.length !== 0}
+        className={styles.resetButton}
```
Comment (layershifter) [inline packages/react-components/react-tags-preview/stories/Tag/TagDismiss.stories.tsx:75]:
```suggestion
        className={styles.resetButton}
        size="small"
```

File: packages/react-components/react-tags-preview/stories/TagGroup/TagGroupDismiss.stories.tsx [hunk 5]
```diff
@@ -7,28 +7,82 @@ import {
...
+              {tag.children}
+            </Tag>
+          ))}
+        </TagGroup>
+      )}
+      <Button
+        onClick={resetItems}
+        ref={resetButtonRef}
+        disabled={visibleTags.length !== 0}
+        className={styles.resetButton}
```
Comment (layershifter) [inline packages/react-components/react-tags-preview/stories/TagGroup/TagGroupDismiss.stories.tsx:81]:
```suggestion
        className={styles.resetButton}
        size="small"
```

File: packages/react-components/react-tags-preview/stories/TagGroup/TagGroupDismiss.stories.tsx [hunk 6]
```diff
@@ -7,28 +7,82 @@ import {
...
+      >
+        Reset Dismiss Tag Example
```
Comment (layershifter) [inline packages/react-components/react-tags-preview/stories/TagGroup/TagGroupDismiss.stories.tsx:83]:
```suggestion
        Reset the example
```

File: packages/react-components/react-tags-preview/stories/TagGroup/TagGroupDismiss.stories.tsx [hunk 7]
```diff
@@ -37,37 +91,45 @@ const DismissWithInteractionTags = () => {
...
+              </InteractionTag>
+            );
+          })}
+        </TagGroup>
+      )}
+      <Button
+        onClick={resetItems}
+        ref={resetButtonRef}
+        disabled={visibleTags.length !== 0}
+        className={styles.resetButton}
```
Comment (layershifter) [inline packages/react-components/react-tags-preview/stories/TagGroup/TagGroupDismiss.stories.tsx:125]:
```suggestion
        className={styles.resetButton}
        size="small"
```

**Review by layershifter — APPROVED** [review]

**Pull request merged**